tests/management/test_querysets.py

```diff
@@ -351,6 +361,47 @@ def test_get_policy_queryset_bad_scope(self):
         with self.assertRaises(serializers.ValidationError):
             get_policy_queryset(req)
 
+    def test_get_access_queryset_org_admin(self):
+        """Test get_access_queryset with an org admin user"""
+        user_data = {"username": "test_user", "email": "admin@example.com"}
+        customer = {"account_id": "10001"}
+        request_context = IdentityRequest._create_request_context(customer, user_data, is_org_admin=True)
+        encoded_req = request_context["request"]
+
+        self._setup_group_for_org_admin_tests()
+
+        user = Mock(spec=User, account="00001", username="test_user")
+        req = Mock(user=user, method="GET", tenant=self.tenant, query_params={APPLICATION_KEY: "app"})
+        req.META = encoded_req.META
+
+        queryset = get_access_queryset(req)
+        self.assertEquals(queryset.count(), 1)
+
+    def test_get_access_queryset_non_org_admin(self):
```

Comment:
Can we also add an explicit test case for a non admin who has RBAC admin access via `rbac:*:*` for this AC?

> As a **non** org-admin who has RBAC admin access via the "User Access administrator" role, when I list my permissions on /api/rbac/v1/access/?application= I should **not** see the access from any "admin_default" roles listed.

Reply:
I tested manually locally and it works as expected, but would be good to have unit test coverage on that case specifically. 😄

Reply:
I am testing it now, just to be clear if I set the permission to `rbac:*:*` that is enough to consider it an RBAC admin? It's working as expected but just want to make sure I have the test setup correctly.

Reply:
Yes, that's correct. The funky thing is that the `rbac:*:*` permission is in the admin default group locally, so you'll still see it in your permissions regardless. What I did was changed the admin default role to have a different permission and tested with that.